pages/apis/graphql/schemas/input_object/pipelinecreateinput.md

```diff
@@ -33,6 +33,6 @@ Autogenerated input type of PipelineCreate
lass="pill pill--scalar pill--normal-case pill--medium" title="Go to SCALAR Int"><code>Int</code></a></p><p>The initial build number to use in the sequence</p></td></tr><tr><td><p><strong><code>organizationId</code></strong><a href="/docs/apis/graphql/schemas/scalar/id" class="pill pill--scalar pill--normal-case pill--medium" title="Go to SCALAR ID"><code>ID!</code></a></p></td></tr><tr><td><p><strong><code>pipelineTemplateId</code></strong><a href="/docs/apis/graphql/schemas/scalar/id" class="pill pill--scalar pill--normal-case pill--medium" title="Go to SCALAR ID"><code>ID</code></a></p><p>The GraphQL ID of the pipeline template you want to use for the pipeline</p></td></tr><tr><td><p><strong><code>repository</code></strong><a href="/docs/apis/graphql/schemas/input_object/pipelinerepositoryinput" class="pill pill--input_object pill--normal-case pill--medium" title="Go to INPUT_OBJECT PipelineRepositoryInput"><code>PipelineRepositoryInput!</code></a></p><p>A source code repository that will back this pipeline</p></td></tr><tr><td><p><strong><code>skipIntermediateBuilds</code></strong><a href="/docs/apis/graphql/schemas/scalar/boolean" class="pill pill--scalar pill--normal-case pill--medium" title="Go to SCALAR Boolean"><code>Boolean</code></a></p><p>If intermediate builds should be skipped as new builds are created</p></td></tr><tr><td><p><strong><code>skipIntermediateBuildsBranchFilter</code></strong><a href="/docs/apis/graphql/schemas/scalar/string" class="pill pill--scalar pill--normal-case pill--medium" title="Go to SCALAR String"><code>String</code></a></p><p>A branch filter to scope which builds this is applied to</p></td></tr><tr><td><p><strong><code>steps</code></strong><a href="/docs/apis/graphql/schemas/input_object/pipelinestepsinput" class="pill pill--input_object pill--normal-case pill--medium" title="Go to INPUT_OBJECT PipelineStepsInput"><code>PipelineStepsInput</code></a></p><p>Steps used by this pipeline defined as YAML</p></td></tr><tr><td><p><strong><code>tags</code></strong><a href="/docs/apis/graphql/schemas/input_object/pipelinetaginput" class="pill pill--input_object pill--normal-case pill--medium" title="Go to INPUT_OBJECT PipelineTagInput"><code>[PipelineTagInput!]</code></a></p><p>Tags you want this pipeline to have</p></td></tr><tr><td><p><strong><code>teams</code></strong><a href="/docs/apis/graphql/schemas/input_object/pipelineteamassignmentinput" class="pill pill--input_object pill--normal-case pill--medium" title="Go to INPUT_OBJECT PipelineTeamAssignmentInput"><code>[PipelineTeamAssignmentInput!]</code></a></p><p>Which teams this pipeline should be assigned to</p></td></tr><tr><td><p><strong><code>visibility</code></strong><a href="/docs/apis/graphql/schemas/enum/pipelinevisibility" class="pill pill--enum pill--normal-case pill--medium" title="Go to ENUM PipelineVisibility"><code>PipelineVisibility</code></a></p><p>The visibility of the pipeline, either PUBLIC or PRIVATE.</p><p>Default value: PRIVATE</p></td></tr>
```

Comment:
I manually diffed this and the only change in here is the build retention fields, and they do not make any mention of the deprecation, so I think these changes should be excluded.

Here are the added lines formatted for clarity:
```html

<tr>
  <td>
    <p><strong><code>buildRetentionEnabled</code></strong><a href="/docs/apis/graphql/schemas/scalar/boolean"
        class="pill pill--scalar pill--normal-case pill--medium" title="Go to SCALAR Boolean"><code>Boolean</code></a>
    </p>
    <p>Choose to keep builds or remove them after a set time period. Pipelines are scanned once a day for builds that
      can be removed according to these settings. Some billing plans require this setting to be enabled.</p>
  </td>
</tr>
<tr>
  <td>
    <p><strong><code>buildRetentionNumber</code></strong><a href="/docs/apis/graphql/schemas/scalar/int"
        class="pill pill--scalar pill--normal-case pill--medium" title="Go to SCALAR Int"><code>Int</code></a></p>
    <p>The minimum number of builds to keep in the pipeline regardless of how old the builds are.</p>
  </td>
</tr>
<tr>
  <td>
    <p><strong><code>buildRetentionPeriod</code></strong><a href="/docs/apis/graphql/schemas/enum/buildretentionperiods"
        class="pill pill--enum pill--normal-case pill--medium"
        title="Go to ENUM BuildRetentionPeriods"><code>BuildRetentionPeriods</code></a></p>
    <p>How long is a build kept before it is automatically removed.</p>
  </td>
</tr>
```

Reply:
Fixed in [846c62a](https://github.com/buildkite/docs/pull/2479/commits/846c62a7731caa00a5ff15353443544e94deb59f)